packages/webapp/src/components/Animals/RemoveAnimalsModal/styles.module.scss

```diff
@@ -0,0 +1,123 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+form > * {
+  margin-top: 24px;
+
+  &:first-child {
+    margin-top: 0;
+  }
+}
+
+fieldset {
+  all: unset;
+  width: 100%;
+}
+
+.mobileOption {
+  display: flex;
+  cursor: pointer;
+  margin-top: 4px;
+  height: 40px;
+
+  input {
+    appearance: none;
+  }
+
+  &Label {
+    height: 100%;
+    width: 100%;
+    border-radius: 8px;
+    padding: 8px 16px;
+  }
+
+  &LabelGreen {
+    border: 1px solid var(--teal600);
```

Comment:
Ah sorry, we should have been more explicit about this this!

We are moving towards a new design system with its [own colour palette](https://www.figma.com/file/eMffC0h3JPlWtzNttxJpil/Litefarm-Updated?node-id=1%3A60&mode=dev). So the Figma colour  `--Colors-Primary-Primary-teal-600;` isn't actually the same colour as `--teal600`.

We use the Figma colours as given, and add any new ones into [src/assets/colors.scss](https://github.com/LiteFarmOrg/LiteFarm/blob/integration/packages/webapp/src/assets/colors.scss) in the section at the bottom for new design system colours. One hesitation I have about the teal in particular, and I'm interested in @antsgar's thoughts, is that I think the React Select's hover effect green is from the original colour system, so maybe there would be a bit of clash? I don't know if you tried it @navDhammu and thought it didn't look good?

But the other colours (the lighter grey colour for the information text and input labels, the reds, browns, etc) -- and probably the greens and teals as well -- should please be kept exactly as written in Figma 🙏

Reply:
Seems the hover effect on the options should now be a different green, `Secondary-green-100` with a value of `#D5EEE1`. I'd add that color as well and update React Select so that it uses the new color for hover throughout